src/svg-loader-components/audio.js

```diff
@@ -1,11 +1,9 @@
-import React from 'react'
-import image from './audio.svg'
-import SVGInline from 'react-svg-inline'
+import React from 'react';
+import SVGInline from 'react-svg-inline';
+import image from './audio.svg';
 
-export class Audio extends React.Component {
-  render() {
-    return(
-      <SVGInline svg={image} {...this.props} />
-    )
-  }
-}
+const Audio = props => (
+  <SVGInline svg={image} {...props} />
+)
+
+export default Audio;
```

Comment:
So for all the exports in the ```svg-loaders-react.js``` "manifest" file (for lack of a better term) to work, the exports themselves can't be default exports.

Can you convert all these exports to ```export { <functional component name> };```?

Reply:
@ajwann I was planning to setup ESLint as well for this project once this PR is merged. Since ESLint prefers using default exports (See https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md), I used this pattern. I can go ahead and convert them like you said.

Reply:
It's definitely not a bad pattern, it just won't work with the way the index.js file does it's exports.
I structured the exports in the index.js file in such a way that users can export all the components in a namespaced style, like so:
```import * as Loaders from 'svg-loaders-react'```

That gives users access to all the loaders by calling them like so: ```<Loaders.Puff />```.

Reply:
Ah I see. Thanks for clearing that up.